frontend/src/store/modules/visitors/actions.js

```diff
@@ -40,6 +46,36 @@ export default {
             context.commit(RESET_BUTTON_FETCHING, data.button);
         }
     },
+    [CHANGE_FETCHED_LINE_CHART_STATE]: (context, value) => {
+
+        if (value) {
+            context.commit(SET_LINE_CHART_FETCHING);
+        } else {
+            context.commit(RESET_LINE_CHART_FETCHING);
+        }
+    },
+    [FETCH_LINE_CHART_DATA]: (context, data) => {
+        if (!data.value) {
+            return;
+        }
+        context.commit(SET_LINE_CHART_FETCHING);
+        const period = getTimeByPeriod(context.state.selectedPeriod);
+        const startDate = period.startDate;
+        const endDate = period.endDate;
+        let chartData = [];
+
+        return factoryVisitorsService.create(context.state.activeButton)
+            .fetchChartValues(startDate.unix(), endDate.unix(), data.groupedParameter)
+                .then(response => {
+                    chartData.items = response.data;
```

Comment:
just for future - Array and Object it's to different stuff, if you want to set attributes in variable, don't define it as array